components/link/link.js

```diff
@@ -115,6 +122,15 @@ class Link extends FocusMixin(LitElement) {
 					overflow: hidden;
 					overflow-wrap: anywhere;
 				}
+				d2l-icon.d2l-new-window {
+					color: var(--d2l-color-celestine);
+					vertical-align: inherit;
+				}
+
+				d2l-icon.d2l-new-window.d2l-icon-small {
```

Comment:
CSS attribute selector here should remove the need for `classMap`:

```suggestion
				:host([small]) d2l-icon.d2l-new-window {
```